vite-plugin-ldaf-icon/index.ts

```diff
@@ -10,7 +10,7 @@ const vitePluginLDAFIcon = async () => {
     const iconPathWithinImg = iconPath.replace(/^node_modules\/@uswds\/uswds\/dist\/img\//, "");
     return [path.basename(iconPath).split(".")[0].replace(/^.+\//, ""), iconPathWithinImg];
   });
-  const iconNames = iconNamesAndPathsWithinImgDir.map(([name]) => name);
+  const iconNames = iconNamesAndPathsWithinImgDir.map(([name]) => name).sort();
```

Comment:
Does the sort order matter?

Reply:
The only reason for this was to make it easier to find icon names in the Storybook stories for the Icon (which is the only place that imports the list of icon names). We could move the sorting to Storybook if you think it'd fit better there.

Reply:
Oh okay that makes sense. No changes needed, I was just curious.